simplq/src/components/ErrorHandler.jsx

```diff
@@ -25,7 +22,7 @@ export class ErrorBoundary extends React.Component {
     // You can also log the error to an error reporting service
     // logErrorToMyService(error, errorInfo);
     console.log('In componentDidCatch');
-    store.dispatch(setErrorNotifOpen(true));
+    store.dispatch(setErrorPopupMessage(true));
```

Comment:
Wanted to understand, why do we set this to "true" here?

Reply:
We are not using `error` and `errorInfo` here? Or maybe these should not be a popul, we should let them go down and be caught by sentry?

Reply:
What I remember from when I wrote this is that the code never reaches this point. That's probably why I didn't notice this was set to true. I'll try to figure this out soon.

Reply:
Do you want to merge this now and use this also for the prod branch?